### DISCUSSION TITLE
Is it possible to change the adminstrator?

### DISCUSSION THREAD

**Discussion by jagjordi** [immich-app/immich#1856] [Q&A]
Hello,
Is it possible to change the administrator user? I.e. assign another user as administrator

**Comment by alextran1502** [comment]
Not yet at the moment; you can do that in the database for now. The plan in the future is to allow more than 1 admin

**Reply by jagjordi** [reply]
Could you give an example on how to do it in the database? Thanks!

**Reply by jrasm91** [reply]
The user table has an isAdmin column, so you just need to connect and set that to false. Afterwards you should be prompted to setup an new admin when you go to the start page.
